app/src/main/java/com/example/mylittlelibrary/service/BookService.kt

```diff
@@ -10,9 +10,17 @@ import retrofit2.Response
 import retrofit2.awaitResponse
 import retrofit2.create
 
-class BookService {
-    private val service = RetrofitClientInstance.retrofitInstance?.create(LibraryApi::class.java)
-    suspend fun fetchBooks(): List<Book>? {
+interface IBookService {
+    val service: LibraryApi?
+
+    suspend fun fetchBooks(): List<Book>?
+
```

Comment:
Create Interface for BookService. URL:https://docs.sonarqube.org/display/SONARQUBE45/Developers%27+Seven+Deadly+Sins